.appveyor/install.cmd

```diff
@@ -1,5 +1,9 @@
 REM Windows install script
 
+rmdir /s /q c:\MinGW
+curl -LO https://github.com/mikee47/SmingTools/releases/download/1.0/MinGW.7z
```

Comment:
@mikee47 Can you add the MinGW toolchain to https://github.com/attachix/sming-toolchains? Hopefully one day we can move all toolchains to SmingHub.

Reply:
OK, I've imported the repo to Attachix, rebased and changed the link.

Reply:
Looks like it didn't get the release properly. I'll recreate it.

Reply:
OK, should be fixed now.